src/init/mountEndpoints.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express';
+
+import { Payload } from '../index';
+import { Endpoint } from '../config/types';
+
+function mountEndpoints(router: Router, endpoints: Endpoint[], ctx: Payload): void {
+  endpoints.forEach((endpoint) => {
+    endpoint.handlers.forEach((handler) => {
```

Comment:
To simplify, can we just do `router[endpoint.method](endpoint.route, endpoints)`?

Reply:
Sure! `.call` is merely personal preference due to readability.